x/nft/go.mod

```diff
@@ -149,6 +154,15 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+// Here are the short-lived replace from the module.
+// Replace here are pending PRs, or version to be tagged.
+replace (
```

Comment:
Did you try using a go.work file in dev so that we don't end up committing a replace directive like this?

Reply:
No. I'm tagging along with what's been pushed to main and updated that.